main.go

```diff
@@ -86,15 +84,10 @@ func main() {
 		},
 		{
 			Name:   "validate",
-			Usage:  "Ensure a sync will not overwrite generated assets in branches that the configuration.yaml wants you to validate against",
```

Comment:
`sync` has been entirely removed and `validate` has been heavily modified for the new scripts.

The process for validate is now:
- `make charts`
- Check if Git is clean. If not, fail and **leave the assets, charts, and index.yaml behind**
- `git checkout UPSTREAM -- assets charts` + regenerate index.yaml
- Check if Git is clean. If not, fail and **leave the assets, charts, and index.yaml behind**

The nice part of this approach is that it is far less complex and has the added benefit that simply running `make validate` locally is sufficient to update charts, assets, and index.yaml directory to the latest assets / charts from packages and the released branch!

It also requires only a single command to be run on CI.

Reply:
So those bullets listed above are the steps performed when `make validate` is run ?

Reply:
Yes